api/topology/src/main/java/quarks/topology/TWindow.java

```diff
@@ -105,8 +129,85 @@ Licensed to the Apache Software Foundation (ASF) under one
      * @return A stream that contains the latest aggregations of partitions in this window.
      */
     <U> TStream<U> batch(BiFunction<List<T>, K, U> batcher);
+
+    /**
+     * Declares a stream that is a continuous, sliding, 
+     * timer triggered aggregation of
+     * partitions in this window.
+     * <P>
+     * Periodically trigger an invocation of
+     * {@code aggregator.apply(tuples, key)}, where {@code tuples} is
+     * a {@code List<T>} containing all the tuples in the partition in
+     * insertion order from oldest to newest.  The list is stable
+     * during the aggregator invocation.  
+     * The list will be empty if the partition is empty.
+     * </P>
+     * <P> 
+     * A non-null {@code aggregator} result is added to the returned stream.
+     * </P>
+     * <P>
+     * Thus the returned stream will contain a sequence of tuples where the
+     * most recent tuple represents the most up to date aggregation of a
+     * partition.
+     *
+     * @param <U> Tuple type
+     * @param period how often to invoke the aggregator
+     * @param unit TimeUnit for {@code period}
+     * @param aggregator
+     *            Logic to aggregation a partition.
+     * @return A stream that contains the latest aggregations of partitions in this window.
+     * 
+     * @see #aggregate(BiFunction)
+     */
+    <U> TStream<U> timedAggregate(long period, TimeUnit unit, BiFunction<List<T>, K, U> aggregator);
```

Comment:
... per-partition.  Sounds like that separate operation could be a count(1)-timedAggregate-evenIfDidntChange.  Does it make sense to force such a user to have to use a timed-trigger-if-changed window followed by this separate operation rather than just use a count(N)-timedAggregate-evenIfDidntChange? (I can imagine it would be ok, just want to be sure)